docs/people-and-groups/saml-okta.md

```diff
@@ -6,26 +6,64 @@ title: SAML with Okta
-| SAML Application Name               | Audience URI (SP Entity ID)                                                                                                  |
+You can also optionally [configure group mappings](#configure-group-mappings) to automatically assign Okta users to Metabase groups.
 
-### Setting attribute statements
+See [authenticating with SAML](./authenticating-with-saml.md) for general SAML info.
 
-Metabase needs you to set the first name (given name), last name (surname), and email address attributes in Okta. These attributes will be passed from Okta to Metabase during authentication to automatically log people into their Metabase accounts.
+## Turn on SAML-based SSO in Metabase
 
-Fill out the **Attribute statements (optional)** section in Okta using the information from your Metabase **SAML attributes** (found under **Admin panel** > **Authentication** > **SAML**).
+In the **Admin**>**Settings** section of the Admin area, go to the **Authentication** tab and click on **Set up** under **SAML**.
+
+You'll see a SAML configuration form like this:
+
+![SAML form](images/saml-form.png)
+
+You'll need to use the information in this form to set up SAML in Okta.
+
+## Set up SAML in Okta
+
+Before configuring SAML authentication in Metabase, you'll need to create and configure a new SAML app integration in Okta.
+
+### Create an app integration
+
+From the Okta **Admin** console, [create a new SAML app integration][okta-saml-docs] to use with Metabase.
+
+### Configure Okta SAML settings
+
+To configure Okta app integration with Metabase, you'll need to use the information in Metabase **Admin panel** > **Authentication** > **SAML**.
+
+#### General settings
+
+| Okta SAML                       | Metabase SAML                                                                                                                               |
+| ------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------- |
+| **Single sign-on URL**          | **URL the IdP should redirect to**. This is your Metabase [Site URL][site-url] -- it should start with `https://` and end with `/auth/sso`. |
+| **Audience URI (SP Entity ID)** | **SAML Application Name** ("Metabase" by default)                                                                                           |
+
+#### Attribute statements
+
+Metabase requires you to set some attributes in Okta:
+
+- email address
+- first name (given name)
+- last name (surname)
+
+These attributes will be passed from Okta to Metabase during authentication to automatically log people into their Metabase accounts.
+
+In **Attribute statements (optional)** section of the Okta application SAML setting, create attribute statements for the email, first name, and last name.
```

Comment:
```suggestion
In the **Attribute statements (optional)** section of the Okta application SAML setting, create the following attribute statements:

- email address
- first name (given name)
- last name (surname)

Even though Okta says these are optional, Metabase requires them. Okta will pass these attributes to Metabase during authentication to automatically log people in to Metabase.
```

The original phrasing (which repeats the attributes) could be confusing given the conflict between "requires" and "optional". The rephrasing omits the repetition of attributes and explains the requirement.